src/01-Learning-JS/04-conditionals.md

```diff
@@ -352,25 +386,38 @@ let person1 = {
 
 It should print out the laborer profession.
 
-```javascript
+```js
 person1['profession'] = "laborer"
 
-// Your code here {
+if(person1['profession'] == 'married') {
   console.log('Person is married.')
 }
-// Your code here {
+if(person1['profession'] == 'laborer') {
   console.log('Person is a laborer.')
 }
-// Your code here {
+if(person1['profession'] == 'widow'){
   console.log('Person is a widow.')
 }
-// Your code here {
+if(person1['profession'] == 'unknown profession'){
```

Comment:
We could instead simply use an `else` statement, if there are no other values to compare.